### PR TITLE
LPS-134354 Add tests for TranslationAdminSelector

### PR DESCRIPTION
**Pull request by carloslancha**
Followup on https://github.com/liferay-frontend/liferay-portal/pull/1372

Added tests for Translation Admin Selector

Fixed an issue about current selected locale not being updated to the default one when removed from the list of active locales.

Test plan:

Is not implemented so you'd need to create a component to use it and call it from some jsp. Here's a component sample: https://gist.github.com/carloslancha/6d7b334d73fd154899f67b678046ae62

~Note: Only two last commits are new, waiting for liferay-portal central repo to sync with Brian's latest master.~

### CODE REVIEW

**Comment by carloslancha** [comment]
ci:stop

File: modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.js [hunk 1]
```diff
@@ -58,6 +58,9 @@ const TranslationAdminSelector = ({
 	);
 
 	const handleCloseTranslationModal = (activeLanguageIds) => {
+		if (!activeLanguageIds.includes(selectedLanguageId)) {
```
Comment (julien) [inline modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/translation_manager/TranslationAdminSelector.js:61]: Make sense, thanks for the fix @carloslancha

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:relevant

**Review by julien — APPROVED** [review]

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
ci:stop

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**